.deploy/accessibility-app/entrypoint.sh

```diff
@@ -22,8 +22,9 @@ then
 
   touch $FILES_PATH/../deploy.lock
 
-  php artisan deploy:initial
-  
+  php artisan deploy:shared
+  php artisan deploy:unique
```

Comment:
@greatislander  php artisan deploy:unique needs to be below line: 30, whatever is within "if [ ! -f $FILES_PATH/../deploy.lock ]" if statement will only run on one webhead